_posts/2021-03-29-Michelle Obama in The Daily Princetonian.md

```diff
@@ -0,0 +1,172 @@
+
+        <break continue="1">(Continued on page three)</break>
+      </column>
+
+      <media>
+        <centered>
+          <photo>...</photo>
+        </centered>
+        <mediacredit>
+          <name>Danny Shapiro</name> — Princetonian
+        </mediacredit>
+
+        <caption>
+          <student>
+            <name>Michelle Robinson</name>
+            <class year="1985">'85</class>
+          </student> models one of the outfits displayed Saturday night in a fashion show in
+          <place type="campus hall">Wu Hall</place> to aid Ethiopian famine victims.
+        </caption>
+      </media>
+
+    </border></item></column>
+  </page>
+
+  <page num="3">
+    <column num="1"><item num="3">
+      <headline>Fashion</headline>
+
+      <break resume="1">(Continued from page one)</break>
+
+      <p cont="true">beat styles. They fashioned designs that highlighted African and Chinese motifs, as well as dashing turbans, sashes, slits, and off-the-shoulder and sleeveless dresses. The creations varied in length but the designers seemed to favor the long and streamlined look.</p>
+
+      <header>Stand-outs</header>
+
+      <p>Some of the designs which stood out as highlights of the show included
+        <nickname>Trish</nickname>'s dresses of African batiste and her terry-cloth outfits,
+        <firstname>Yvonne</firstname>'s dark sleeveless dress dramatized by contrasting sashes and
+        <firstname>Karen</firstname>'s romantic organza skirt and lace top combination.
+      </p>
+
+      <p>Members of the audience expressed enthusiasm for the students' fashions, and gave the three designers a standing ovation at the end.
+        <quote>The presentation
+          <quotation>"gathered up a bunch of dynamic personalities, dynamic models — an all-over dynamic show,"</quotation> said
+          <quoted>
+            <student>
+              <name>Eugene Vanhorn</name>
+              <class year="1988">'88.</class>
+            </student>
+          </quoted>
+        </quote>
+      </p>
+
+      <p>
+        <quote>
+          <quotation>"I was really surprised at the very unique ideas expressed,"</quotation> added
+          <quoted>
+            <student>
+              <name>Lisa Brooks</name>
+              <class year="1988">'88</class>
+            </student>
+          </quoted>.
+        </quote>
+      </p>
+    </item></column>
+  </page>
+</div>
+```
+
+### Michelle Obama's Photo ###
+And for the reader who has made it this far, here is the photo from the encoded article that shows Obama modeling at a charity fashion show during her senior year.
+
+![Michelle Obama Fashion Show](https://github.com/jpgarcia2023/hello-world/blob/main/Princetonian_1985-02-26_v109_n018_0001%20photo.png?raw=true)
```

Comment:
Since this image is really large, why don't you add a `{:width="500px"}` parameter to shrink it a bit just after the parentheses giving the link to the image.